lib/src/backend/declarer.dart

```diff
@@ -111,7 +118,7 @@ class Declarer {
         skip: skip,
         onPlatform: onPlatform,
         tags: tags,
-        retry: retry));
+        retry: _doRetry ? retry : 0));
```

Comment:
We need to explicitly set 0 to override annotations. I have added a corresponding test.